src/Data/CurrentUser/CurrentUserDataItem.ts

```diff
@@ -97,6 +102,8 @@ export const CurrentUser = provideDataItem('CurrentUser', {
       }
     },
     async update(params) {
+      if (getPisaAuthorization()) throw new Error('Update not supported.')
```

Comment:
* [ ] FYI Not sure if `DataConnectionError` is the better choice for such cases?